ui/package.json

```diff
@@ -96,7 +94,6 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-simple-import-sort": "^8.0.0",
     "eslint-plugin-sort-keys-fix": "^1.1.2",
-    "lolex": "^3.0.0",
```

Comment:
This is now FakeTimers, which is pulled in transitively by various Jest packages.  No need to list it here.